src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -129,20 +176,51 @@ export class MarkdownToolbarComponent {
 			{ action: this._toggleSplitViewAction },
 			{ action: this._toggleMarkdownViewAction }
 		];
+
+		this._previewModeTaskbarContent = [
+			{ action: boldButton },
+			{ action: italicButton },
+			{ action: underlineButton },
+			{ action: highlightButton },
+			{ action: codeButton },
+			{ element: linkButtonContainer },
+			{ action: listButton },
+			{ action: orderedListButton },
+			{ element: imageButtonContainer },
+			{ element: buttonDropdownContainer },
+			{ action: this._toggleTextViewAction },
+			{ action: this._toggleSplitViewAction },
+			{ action: this._toggleMarkdownViewAction }
+		];
+
 		// Hide link and image buttons in WYSIWYG mode
 		if (this.cellModel.showPreview && !this.cellModel.showMarkdown) {
 			this._actionBar.setContent(this._wysiwygTaskbarContent);
 		} else {
-			this._actionBar.setContent(this._taskbarContent);
+			if (this.previewFeaturesEnabled) {
+				this._actionBar.setContent(this._previewModeTaskbarContent);
+			} else {
+				this._actionBar.setContent(this._taskbarContent);
+			}
 		}
 	}
 
+	public onCustomButtonClick(event, type: MarkdownButtonType) {
```

Comment:
Give the event a type and the function a return type

Reply:
Note: I used event type: `any` instead of `MouseEvent`, otherwise I could not get event.target.

Reply:
What do you mean you couldn't "get" event.target? That should be a property on the MouseEvent

Reply:
Problem solved.